public/docs/getting_started/client.md

```diff
@@ -1,20 +1,54 @@
+# Choosing an Ethereum client
+
 There are many Ethereum clients to choose from. We recommend using different clients when developing and deploying.
 
-# When Developing
+## Developing locally
+
+### Truffle Develop
+
+We recommend using Truffle Develop, an Ethereum client built directly into Truffle which contains an in-memory blockchain that runs only on your development machine.
+
+It processes transactions instantly instead of waiting for the default block time, so you can test that your code works quickly, and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing.
```

Comment:
New paragraph; should probably use "Truffle Develop" instead of It"

Reply:
I see you tried to pare down what was written for the TestRPC, but this effectively removed a lot of "the sell". i.e., we need to convince people to use it, and if you compare the new description of the TestRPC to this description of Truffle Develop, the TestRPC is sold a lot better. We should take more from the TestTPC's "sell" as we want to encourage people to use Truffle Develop instead. Here's my attempt at merging the two. Note that "in memory blockchain" is no longer correct, so I removed that.

> We recommend using Truffle Develop, a development blockchain built directly into Truffle. It processes transactions instantly instead of waiting for the default block time -- so you can test that your code works quickly -- and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing, and Truffle knows how to use its special features to speed up test runtime by almost 90% compared to other clients. 

See my notes below about how to deemphasize the TestRPC.

Reply:
Thanks for the rewrite, that's very helpful.

Can you provide clarification about the "in-memory blockchain" not being correct? I took that right from @gnidan 's oral description, and I've been using that phrase pretty much everywhere (so if it's not correct, I'll want to change it in a bunch of places).

Reply:
As of TestRPC 4.0, the blockchain is no longer stored in memory, as the blockchain takes up a lot of data, which effectively makes it a built in memory leak (not good). Instead, it's now stored on the filesystem and has a database attached to it. For now, each launch of the TestRPC creates a temp directory to store that data to maintain the same behavior it hand when it was initially created.